MIP/mip-103/roadmap/primitive/README.md

```diff
@@ -0,0 +1,13 @@
+# Primitive
+The Primitive stage of the roadmap is focused on the development and testing of implementations of FFS which are decoupled from Movement Lab's full node logic or any particular VM. This primarily means cleaning up existing proof of concept implementations, proving their correctness, and designing a more modular API. The initial implementation [Baker Confirmations (ZK-FFS)](../README.md) also begins under this stage.
+
+## Key Deliverables
+
+### contract-based `ffs`
+1. MCR
```

Comment:
should start to think of a suitable name, rather than Multi-Commit-Rollup. 

- PCP ( PostConfirmation Protocol)
- AP (Anchoring Protocol) 
- APP (Anchoring Postconfirmation Protocol)